src/app/doctor-form/doctor-form.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { Doctor } from "../models/doctor";
+
+@Component({
+  selector: 'app-doctor-form',
+  templateUrl: './doctor-form.component.html',
+  styleUrls: ['./doctor-form.component.scss']
+})
+export class DoctorFormComponent implements OnInit {
+
+  ngOnInit(): void {
+  }
+
+  model = new Doctor(18, '', '', '');
```

Comment:
please be consistent when you name variable, use doctor instead as you manipulate Doctor entity